terraform/azure/pipeline/target_snowflake.tf

```diff
@@ -0,0 +1,95 @@
+module "sf_message_queue_eh_topic" {
+  source  = "snowplow-devops/event-hub/azurerm"
+  version = "0.1.1"
+
+  count = var.snowflake_enabled ? 1 : 0
+
+  name                = "snowflake-loader-topic"
+  namespace_name      = module.eh_namespace.name
+  resource_group_name = var.resource_group_name
+}
+
+module "sf_transformer_storage_container" {
```

Comment:
I would argue that this belongs in the pipeline file, or that both the account and the container belong in this stack (or probably factored differently to avoid duplication but not the snowflake-specific file).

Reply:
You can have multiple containers in a single storage account though can't you?  Only the parts specific to the loader are in this file was the rationale.

Reply:
Yes you can, but we're not just using any random storage account, we're using one we specifically create for this stack, and this stack doesn't support multiple loaders. So the storage account is specific to then loader in the context of this stack. 

It doesn't make a real difference, I can see the logic in how it's done, just prodding at it.